CMakeLists.txt

```diff
@@ -1,18 +1,70 @@
-cmake_minimum_required(VERSION 3.10)
+# Shamelessly Ripped from llvm tutor
```

Comment:
I presume you mean https://github.com/banach-space/llvm-tutor? When I do things like this, I usually put a permalink to the specific file/commit that I borrowed from.

Reply:
(Sorry about the passive aggressive tone. Please add the comment I'm suggesting here.)